examples/elevenlabs/text-to-speech.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\AI\Platform\Bridge\ElevenLabs\ElevenLabs;
+use Symfony\AI\Platform\Bridge\ElevenLabs\PlatformFactory;
+use Symfony\AI\Platform\Message\Content\Text;
+
+require_once dirname(__DIR__).'/bootstrap.php';
+
+$platform = PlatformFactory::create(
+    apiKey: env('ELEVEN_LABS_API_KEY'),
+    outputPath: dirname(__DIR__).'/tmp',
+    httpClient: http_client(),
+);
+$model = new ElevenLabs(options: [
+    'voice' => 'Dslrhjl3ZpzrctukrQSN', // Brad (https://elevenlabs.io/app/voice-library?voiceId=Dslrhjl3ZpzrctukrQSN)
+]);
+
+$result = $platform->invoke($model, new Text('Hello world'));
+
+echo $result->asBase64().\PHP_EOL;
```

Comment:
what do you think about enabling this?
```bash
php elevenlabs/text-to-speech.php | mpg123 -
```
that works for me to easily run the example, but requires this:
```suggestion
echo $result->asBinary();
```

(didn't figure out a way to play it from php directly)